### DISCUSSION TITLE
Allow MA to be accessed directly from HA menu

### DISCUSSION THREAD

**Discussion by seancmalloy** [music-assistant/support#1582] [Feature requests and ideas]
i would like to have a way to detach music assistant from HA, perhaps with button/link to open new browser window. this could be configured in web server settings. the link could just be the icon/title or upper toolbar.

![Screenshot (17)](https://github.com/music-assistant/hass-music-assistant/assets/23019226/c218cc58-b7ba-4355-8995-8883151a1f75)

**Comment by OzGav** [comment]
Are you aware you can expose the UI via port 8095. We don’t recommend this for security reasons.

**Comment by seancmalloy** [comment]
Yes. I am currently using nginx proxy to pass ma to ssl connection. and on top of that i only access ha remotely through vpn. 

![Screenshot (18)](https://github.com/music-assistant/hass-music-assistant/assets/23019226/b703435b-2703-4950-9c9e-005fb1c50937)

**Reply by OzGav** [reply]
So what else do you need? You can access MA directly via port 8095 and you could add that to the HA menus?
